server_src/routes/index.js

```diff
@@ -26,8 +26,7 @@ const { fetchMongoDBConnectionString } = require("../../app-env");
 const { isPm2, isNodemon, isNode } = require("../utils/env.utils.js");
 
 const SystemInfo = systemInfo.SystemRunner;
-const version = process.env.npm_package_version;
-console.log(`Version: ${version} (server started)`);
```

Comment:
I dont index.js should produce any logs at startup as it is run soooo many times in tests.

Reply:
No worries, was gonna remove that myself it's legacy now anyway